### ISSUE TITLE
About finetune my own data

### ISSUE THREAD

**Issue by AITTSMD** [sanghoon/pva-faster-rcnn#27]
@sanghoon hi,sorry to bother u,but I have some problem when I finetune my own data.My task is to detect the whale in the image.when I use the full/test.model and models/pvanet/example_finetune/solver.prototxt to finetune.I got a bad result(Why?).Then,I use example_train_384/ to finetune.The result is better,but the detection time is long,about 0.18s/img（gpu gtx1070）.The example_train_384/test.prototxt has bn layers.Is it the reason?Can u give some advice about how to finetune my own data? Thank you in advance!

**Comment by catsdogone** [comment]
@sanghoon 
I finetune the pvanet/full/test.model using the train.prototxt in pvanet/example_fineturn on KITTI while the result is not good and even worse than the pvanet/full/test.model. Should I use imagenet/full/test.model as pre-train model if I want to train on a dataset with different classes with VOC? 
Can you give me some advice? Thank you very much.
There are related files：
[log.txt](https://github.com/sanghoon/pva-faster-rcnn/files/623864/log.txt)
[solver_plateau.txt](https://github.com/sanghoon/pva-faster-rcnn/files/623865/solver_plateau.txt)
[train.txt](https://github.com/sanghoon/pva-faster-rcnn/files/623866/train.txt)
![log txt train_3](https://cloud.githubusercontent.com/assets/7392509/20784829/c6ac7e14-b7d7-11e6-983a-da07887458d4.png)

The ap @IOU0.7:
AP for Car = 0.7321
AP for Van = 0.6796
AP for Truck = 0.8270
AP for Pedestrian = 0.4119
AP for Person_sitting = 0.3100
AP for Tram = 0.6638
AP for Cyclist = 0.4930
Mean AP = 0.5882

**Comment by AITTSMD** [comment]
@catsdogone HI,I wonder how you draw the pictures

**Comment by catsdogone** [comment]
@AITTSMD Using the parse.py in caffe-fast-rcnn/tools/extra to parse the log file than plot it.

**Comment by AITTSMD** [comment]
@catsdogone oh! Thank u,I''ll have a try.By the way,when you have limited data,you can use example_384 to finetune.The result will be better,but the speed will be slower!

**Comment by catsdogone** [comment]
@AITTSMD Thank you. I will have a try.

**Comment by theuy** [comment]
@AITTSMD HI, I wonder which model do you use for the initial weights while you use example_384 to fineturne, models/pvanet/full/test.model ?

**Comment by AITTSMD** [comment]
@catsdogone  imagenet/train.model

**Comment by beihangzxm123** [comment]
@catsdogone Could you please share the details that how you generate the log file and how to figure the plots of loss-iters?

**Comment by sanghoon** [comment]
Hi all,
Sorry for a late comment.
I'm attending a conference right now, I'll check the issue when I get back to my home

One more thing,
We've trained a better model with a similar structure.
It'll be shared soon (maybe in a couple of weeks)

**Comment by catsdogone** [comment]
@beihangzxm123 I use the format of script file as py-faster-rcnn/scripts/*.sh. The parse_log.py is from caffe-fast-rcnn/toos/extra. The attachment is the files you may need.
[kitti_pva.txt](https://github.com/sanghoon/pva-faster-rcnn/files/641313/kitti_pva.txt)--kitti_pva.sh
[parse_log.txt](https://github.com/sanghoon/pva-faster-rcnn/files/641314/parse_log.txt)--parse_log.py
[plot_loss.txt](https://github.com/sanghoon/pva-faster-rcnn/files/641315/plot_loss.txt)--plot_loss.py
